src/Command/Doctrine/AbstractCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Bernard\Command\Doctrine;
+
+use Bernard\Doctrine\MessagesSchema;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\Synchronizer\SingleDatabaseSynchronizer as Synchronizer;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package Bernard
+ */
+abstract class AbstractCommand extends Command
+{
+    protected $name = 'abstract';
+
+    public function __construct()
```

Comment:
require the name in the constructor, this way we wont have any bernard:doctrine:abstracts.